src/entities/MakerBalanceChainCacheEntity.ts

```diff
@@ -0,0 +1,20 @@
+import { BigNumber } from '@0x/utils';
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+import { BigNumberTransformer } from './transformers';
+
+// A table of cached erc20 balances for RFQT market makers
+@Entity({ name: 'maker_balance_chain_cache' })
+export class MakerBalanceChainCache {
+    @PrimaryColumn({ name: 'token_address', type: 'varchar' })
+    public tokenAddress?: string;
+
+    @PrimaryColumn({ name: 'maker_address', type: 'varchar' })
+    public makerAddress?: string;
+
+    @Column({ name: 'balance', type: 'numeric', nullable: true, transformer: BigNumberTransformer })
```

Comment:
oh I see, so we are saving this as a string, and the TS code will "deserialize" it as a Big Int?

Reply:
It TS will read it as a BigNumber, and it's stored as a string. That's how it's done for large values for the meta-tx entities, and I think that's a pretty safe way to handle it.